packages/manager/modules/octavia-load-balancer/src/octavia-load-balancer.module.js

```diff
@@ -26,12 +27,15 @@ angular
     'ui.router',
     ngOvhUtils,
     uiKit,
+    'oui',
     onboarding,
     dashboard,
     create,
-    ListLayoutHelper.moduleName,
+    deleteComponent,
+    loadbalancers,
```

Comment:
```suggestion
    delete,
    loadBalancers,
```

Reply:
Hello, "delete" is a reserved javascript keyword, that's why I had to use "deleteComponent" instead.